manim/_config/utils.py

```diff
@@ -1,8 +1,10 @@
-"""
-utils.py
---------
+"""Utilities to create and set the config.
 
-Functions to create and set the config.
+The main class exported by this module is :class:`ManimConfig`.  This class
+contains all configuration options, including frame geometry (e.g. frame
+height/width, frame rate), output (e.g. directories, logging), styling
+(e.g. background color, transparency), and general behavior (e.g. writing a
+movie vs writing a single frame).
```

Comment:
I think it makes sense to add a reference to the configuration tutorial here as well.

```suggestion
movie vs writing a single frame).

See :doc:`tutorials/configuration` for an introduction to Manim's configuration system.
```